streaming/src/test/scala/org/apache/spark/streaming/StateMapSuite.scala

```diff
@@ -267,12 +270,17 @@ class StateMapSuite extends SparkFunSuite {
     assertMap(stateMap, refMap.toMap, time, "Final state map does not match reference map")
   }
 
-  private def testSerialization[MapType <: StateMap[Int, Int]](
+  private def testSerialization[MapType <: StateMap[Int, Int] : ClassTag](
     map: MapType, msg: String): MapType = {
-    val deserMap = Utils.deserialize[MapType](
-      Utils.serialize(map), Thread.currentThread().getContextClassLoader)
-    assertMap(deserMap, map, 1, msg)
-    deserMap
+    val deserMaps = Array(new JavaSerializer(conf), new KryoSerializer(conf)).map {
```

Comment:
Easier to read if this is made a function and called with two different serializers.